controllers/competition_supervisor/competition_supervisor.py

```diff
@@ -24,16 +24,22 @@ def recording_path() -> Path:
     date = now.date().isoformat()
     # Windows doesn't like colons in filenames
     name = now.isoformat().replace(':', '-')
-    return Path(date) / name / '{}.html'.format(name)
+    return Path(date) / name / '{}'.format(name)
 
 
 @contextlib.contextmanager
 def record_animation(supervisor: Supervisor, file_path: Path) -> Iterator[None]:
     file_path.parent.mkdir(parents=True, exist_ok=True)
-    print("Saving animation to {}".format(file_path))
-    supervisor.animationStartRecording(str(file_path))
+    print("Saving animation to {}.html".format(file_path))
+    print("Saving video to {}.mp4".format(file_path))
+    supervisor.animationStartRecording(str(file_path) + '.html')
+    supervisor.movieStartRecording(str(file_path) + '.mp4',
```

Comment:
I would suggest that we construct the full names upfront rather than constructing them twice -- that avoids any chance of them getting out of step.